website/src/release.md

```diff
@@ -316,33 +320,33 @@ After downloading them, here are the instructions on how to verify them.
 curl https://downloads.apache.org/iceberg/KEYS -o KEYS
 gpg --import KEYS
 ```
-* Verify the `.asc` file: ```gpg --verify apache-iceberg-rust-${iceberg_version}.tar.gz.asc```
-* Verify the checksums: ```shasum -a 512 -c apache-iceberg-rust-${iceberg_version}.tar.gz.sha512```
-* Verify license headers: ```docker run -it --rm -v $(pwd):/github/workspace apache/skywalking-eyes header check```
+* Verify the `.asc` file: ```gpg --verify apache-iceberg-rust-*.tar.gz.asc```
+* Verify the checksums: ```shasum -a 512 -c apache-iceberg-rust-*.tar.gz.sha512```
 * Verify build and test:
 ```bash
-tar -xzf apache-iceberg-rust-${iceberg_version}.tar.gz
-cd apache-iceberg-rust-${iceberg_version}
+tar -xzf apache-iceberg-rust-*.tar.gz
+cd apache-iceberg-rust-*/
 make build && make test
 ```
+* Verify license headers: ```docker run -it --rm -v $(pwd):/github/workspace apache/skywalking-eyes header check```
```

Comment:
maybe we need to move this step before the build and test? Otherwise we will need to run `make clean` first to clean up files under the `target/`

Reply:
yea good point, it was previous in that order. let me put it back